src/fn.js

```diff
@@ -31,22 +30,18 @@ export default (id, custom) => (
       regex,
       some,
       typeOf,
+      ...custom,
```

Comment:
I think Object.assign might be faster than object spread. I haven't run the tests agains it, but this exists...

https://www.measurethat.net/Benchmarks/Show/2978/0/lodash-merge-vs-objectassign-vs-spread

Not a big deal, but if we're really looking to squeeze some speed out of here it's a thing we could look into.

Reply:
This website is super handy - I'm gonna change it here